katello-agent/katello-agent.spec

```diff
@@ -39,15 +40,35 @@ Requires: yum-plugin-security
 Requires: yum-security
 %endif
 
+%description
+Provides plugin for gofer, which allows communicating with Katello server
+and execute scheduled actions.
+
+%package -n katello-host-tools
+BuildArch: noarch
+Summary: A set of commands and yum plugins that support a katello host
+
+Requires: subscription-manager
```

Comment:
Does this upper level package actually require subscription-manager or just the fact-plugin?